### ISSUE TITLE
Firing minigun doesn't render properly in F5

### ISSUE THREAD

**Issue by ThizThizzyDizzy** [TeamPneumatic/pnc-repressurized#367]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

Minecraft version 1.12.2

Minecraft forge 14.23.5.2838

Mod version 1.12.2-0.11.3-366

![image](https://user-images.githubusercontent.com/22590237/60744489-1e82ec80-9f44-11e9-8e8b-013c77e8e5e1.png)
Steps to reproduce:
1. create a minigun; add ammo to it
2. Press F5 twice
3. Look to the side so your head and body are not aligned
4. Hold right click

**Comment by desht** [comment]
Yeah, a long standing problem that I never managed to fix. Maybe one day!
